packages/lesswrong/components/editor/EditorFormComponent.tsx

```diff
@@ -400,7 +418,7 @@ class EditorFormComponent extends Component<EditorFormComponentProps,EditorFormC
       [fieldName]: data ? {
         originalContents: {type, data},
         commitMessage, updateType,
-      } : undefined
+      } : null
```

Comment:
This change allowed my new form to actually clear CKEditor fields (but not other forms 😢)